apps/etterlatte-beregning/src/main/kotlin/beregning/regler/barnepensjon/BeregnBarnepensjon1967.kt

```diff
@@ -49,4 +56,22 @@ val kroneavrundetBarnepensjonRegel = RegelMeta(
     regelReferanse = RegelReferanse(id = "REGEL-KRONEAVRUNDING")
 ) benytter beregnBarnepensjon1967Regel med { beregnetBarnepensjon ->
     beregnetBarnepensjon.round(decimals = 0).toInteger()
+}
+
+val beregnBarnepensjon1967RegelMedInstitusjon = RegelMeta(
+    gjelderFra = BP_1967_DATO,
+    beskrivelse = "Reduserer ytelsen mot opptjening i folketrygden inkludert institusjonsopphold",
+    regelReferanse = RegelReferanse(id = "BP-BEREGNING-1967-REDUSERMOTTRYGDETID-INSTITUSJON")
+) benytter barnepensjonSatsRegel og trygdetidsFaktor og institusjonsoppholdRegel med { sats,
```

Comment:
Akkurat denne regelen blir nok litt forskjellig, siden det her skal komme inn en ekstra vurdering om institusjonsoppholdregelen gir høyere enn vanlig sats. Det er sannsynligvis nødvendig med en institusjonsoppholdSats i stedet for en faktor, som benytter G direkte, og så sammenligner med det bruker hadde før. Tror heller ikke trygdetidfaktoren er en del av institusjonsoppholdbiten direkte, men dette må avklares litt

Eksempel for å illustrere:
Ytelsen gir vanligvis 8 % av G, på grunn av en kombinasjon av trygdetid og søskenjustering. Saksbehandler velger full reduksjon. Nå skal bruker ha 8 % av G, og ikke 0,8 % eller 10 %